UPGRADE-2.1.md

```
@@ -1,25 +1,311 @@
-# UPGRADE FROM `2.1.9` TO `2.1.10`
+# UPGRADE FROM `2.1.12` TO `2.1.13`
 
-## Redirect behavior changes
+### Telemetry improvements
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)